application/common/models/User.php

```diff
@@ -570,6 +570,7 @@ public function getMethodFields()
         return [
             'add' => $this->getNagState() == self::NAG_ADD_METHOD ? 'yes' : 'no',
             'review' => $this->getNagState() == self::NAG_REVIEW_METHOD ? 'yes' : 'no',
+            'options' => $this->methods,
```

Comment:
We discussed not including the options/methods if they don't need to be reviewed, do you want to update that before we approve/merge this or come back to it?

Reply:
Pushed the update just now.